pkg/asset/agent/installconfig.go

```diff
@@ -297,10 +321,6 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 			fieldPath := field.NewPath("Platform", "Baremetal", "LibvirtURI")
 			logrus.Debugf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.LibvirtURI))
 		}
-		if baremetal.ClusterProvisioningIP != defaultBM.ClusterProvisioningIP {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ClusterProvisioningIP")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ClusterProvisioningIP))
-		}
 		if baremetal.DeprecatedProvisioningHostIP != defaultBM.DeprecatedProvisioningHostIP {
```

Comment:
DeprecatedProvisioningHostIP was replaced by ClusterProvisioningIP, so if we are removing the warning for the latter we should do the same for the former.

Reply:
Good point, removed.